src/cdk/overlay/overlay-ref.ts

```diff
@@ -23,11 +24,14 @@ export class OverlayRef implements PortalHost {
   private _attachments = new Subject<void>();
   private _detachments = new Subject<void>();
 
+  _dispatchedKeydownEvents = new Subject<KeyboardEvent>();
```

Comment:
Add a description to this one. Also potentially rename it to something like `_keydownEvents`.